frontend/src/HomePage/TemplateLibraryModal/TemplateDisplay.jsx

```diff
@@ -48,14 +41,19 @@ export default function TemplateDisplay(props) {
             ))}
           </span>
         </Row>
-        <Row className="align-items-center justify-content-center" style={{ height: '88%' }}>
-          {isloaded && (
-            <LazyLoad>
-              <img
-                className="template-image"
-                src={`/assets/images/templates/${id}${props.darkMode ? '-dark' : ''}.png`}
-              />
-            </LazyLoad>
+        <Row className="align-items-center justify-content-center" style={{ height: '88%', position: 'relative' }}>
+          <LazyLoadImage
```

Comment:
Shall we move this to a reusable component?

Reply:
Are you proposing, a custom component that includes a lazyloading image with spinner functionality, right?